Expecto/Expecto.fs

```diff
@@ -1068,11 +1095,13 @@ module Impl =
         let next = List.length tests |> rand.Next
         List.item next tests
 
-      let finishTimestamp =
-        lazy
+      let totalTicks =
         config.stress.Value.TotalSeconds * float Stopwatch.Frequency
         |> int64
-        |> (+) (Stopwatch.GetTimestamp())
+
+      let finishTime =
+        lazy
+        totalTicks |> (+) (Stopwatch.GetTimestamp())
```

Comment:
Remember that Stopwatch's ticks differs between operating systems and is not the same as DateTime/BCL ticks.